styles.css

```diff
@@ -114,3 +114,417 @@ p {
+  height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  padding: 10px 10px;
+  margin: 12px;
+}
+
+.program-stack2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+.program-stack3 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+.program-stack4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  border: 1px solid #8993a4;
+  flex-direction: row;
+}
+
+/* ----------------------end------------- */
+.professional-art {
+  margin-bottom: 100px;
+}
+
+.position-items {
+  display: flex;
+  gap: 12px;
+}
+
+.programs {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.see-project {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+
+  /* display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em; */
+}
+
+/* backgroud picture and stack */
+
+/* background effect */
+
+.data-arc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  height: 87px;
+  padding-left: 24px;
+  width: 100%;
+}
+
+.see-project-btn {
+  position: relative;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  background: #ff6b00;
+  color: #fff;
+  border: #fff;
+}
+
+.details {
+  background-image: url(images/bg-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  color: white;
+  padding-top: 20%;
+  margin: 35px;
+}
+
+/* about me section */
+
+#about-me-sec {
+  background-color: #1c1a19;
+  color: #fff;
+  align-items: center;
+  text-align: center;
+}
+
+.about-me {
+  background-image: url(images/about-me.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-right: 24px;
+  padding-bottom: 220px;
+  align-items: flex-start;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+}
+
+.get-my-resume-btn {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background: #ff6b00;
+  border: #fff;
+  text-align: left;
+  padding: 12px;
+  padding-left: 20px;
+  margin-left: 25px;
+}
+
+#color {
+  margin-left: 15px;
+  color: #091e42;
+}
+
+.vertical-list,
+.stack-Skills {
+  align-items: center;
+  text-align: center;
+}
+
+/* .about-me {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+} */
```

Comment:
- [ ]  kindly consider removing the unwanted piece of codes they make your codebase look messy and this will help it look more clean

Reply:
removed